### ISSUE TITLE
Create Apple Developer account

### ISSUE THREAD

**Issue by BenjaminStorer** [MufasaTheMusician/eatwithme#20]
Creating Apple developer account for EatWithMe company.

**Comment by BenjaminStorer** [comment]
Currently blocked by the need to update EatWithMe business details.

**Comment by BenjaminStorer** [comment]
Encountered another issue which may need Masaba's input. Creating the Apple ID that will be used requires some personal information that I think Masaba should fill out.
